Sources/Views/InputTextView.swift

```diff
@@ -176,4 +184,84 @@ open class InputTextView: UITextView {
         placeholderLabelConstraintSet?.left?.constant = placeholderLabelInsets.left
         placeholderLabelConstraintSet?.right?.constant = -placeholderLabelInsets.right
     }
+    
+    // MARK: - Image Paste Support
+    
+    open override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
+        
+        if action == NSSelectorFromString("paste:") && UIPasteboard.general.image != nil {
+            return isImagePasteEnabled
+        }
+        return super.canPerformAction(action, withSender: sender)
+    }
+    
+    open override func paste(_ sender: Any?) {
+        
+        guard let image = UIPasteboard.general.image else {
+            return super.paste(sender)
+        }
+        pasteImageInTextContainer(with: image)
+    }
+    
+    /// Addes a new UIImage to the NSTextContainer as an NSTextAttachment
+    ///
+    /// - Parameter image: The image to add
+    private func pasteImageInTextContainer(with image: UIImage) {
+        
+        let font = self.font ?? UIFont.preferredFont(forTextStyle: .body)
+        let textColor = self.textColor ?? .black
+        
+        let attributes: [NSAttributedStringKey:Any] = [
+            NSAttributedStringKey.font : font,
+            NSAttributedStringKey.foregroundColor : textColor,
+        ]
+
+        // Add the new image as an NSTextAttachment
+        let attributedImageString = NSAttributedString(attachment: textAttachment(using: image))
+        
+        let newAttributedStingComponent = NSMutableAttributedString(string: "")
```

Comment:
Maybe we can use `init(attachment: NSTextAttachment)` in here instead of use `append`.